rasa/shared/nlu/training_data/formats/rasa_yaml.py

```diff
@@ -378,6 +378,23 @@ def dump(
             target: Name of the target object to write the YAML to.
             training_data: TrainingData object.
         """
+        result = self.training_data_to_yaml(training_data)
+
+        if result:
+            rasa.shared.utils.io.write_yaml(result, target, True)
+
+    def training_data_to_yaml(
```

Comment:
Thinking out loud - is there actually any reason for this method not to be static?

Reply:
That's true, I've changed it now